src/assets/emptyRight.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="9" height="13" viewBox="0 0 9 13" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I don't think we need these, this is what is causing the weird offset I mentioned on the PR. Lets hide instead.